src/main/java/org/opensearch/neuralsearch/processor/TextChunkingProcessor.java

```diff
@@ -250,8 +255,11 @@ private void chunkMapType(
             } else {
                 // chunk the object when target key is of leaf type (null, string and list of string)
                 Object chunkObject = sourceAndMetadataMap.get(originalKey);
-                List<String> chunkedResult = chunkLeafType(chunkObject, runtimeParameters);
-                sourceAndMetadataMap.put(String.valueOf(targetKey), chunkedResult);
+
+                if (!(ignoreMissing && chunkObject == null)) {
```

Comment:
I don't think this one got addressed. Can you please reply to each comment:
- whether you're making changes or not
- if making change whether you agree with suggestion/request, and if not please put your resoning

Reply:
@martin-gaievski 

In my opinion there is no point in having a method for a 2 variables boolean expression  that is used just one time.

It will convolute the code more than help

Reply:
it's doesn't really matter if it used only once, my point is that it should be easy to tell what this condition means by looking to the code. It will be easy to maintain because this will lower the chance of mistake.
You can put a comment or pull it to the method. I find the method easy to read because you can encapsulate the negation into the method itself, something like folowing: 
```
private boolean shouldProcessChunk(boolean ignoreMissing, Object chunkObject) {
    return !(ignoreMissing && chunkObject == null);
}
```

Reply:
I understand what you say. I still don't fully agree. Either way I u[dated the PR